assets/csidriveroperators/shared-resource/13_config_configmap.yaml

```diff
@@ -0,0 +1,5 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: csi-driver-shared-resource-config
+  namespace: openshift-cluster-csi-drivers
```

Comment:
eventually this will have a `config.yaml` entry similar to https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/manifests/03_configmap.yaml